service/src/main/kotlin/no/nav/su/se/bakover/service/klage/KlageServiceImpl.kt

```diff
@@ -261,18 +283,69 @@ class KlageServiceImpl(
         return oversendtKlage.right()
     }
 
+    override fun avvis(
+        klageId: UUID,
+        attestant: NavIdentBruker.Attestant,
+    ): Either<KunneIkkeIverksetteAvvistKlage, IverksattAvvistKlage> {
+        val klage = klageRepo.hentKlage(klageId) ?: return KunneIkkeIverksetteAvvistKlage.FantIkkeKlage.left()
+
+        val avvistKlage = klage.avvis(
+            Attestering.Iverksatt(
+                attestant = attestant,
+                opprettet = Tidspunkt.now(clock),
```

Comment:
Hvorfor ta en ny timestamp her? Har vi et eget konsept for iverksettelsestidspunktet? (trodde det var vedtaket)

Reply:
Dette er tidspunktet for attesteringen (aka Attesteringshistorikk)